src/main/scala/Dictionary.scala

```diff
@@ -0,0 +1,36 @@
+import java.nio.file.Paths
+
+import scala.io.BufferedSource
+import scala.io.Source._
+
+object Dictionary {
+
+
+  def extractUniqueWordsFromTextFile(filePath: String): Set[String] = {
+    val textFilePath: String = textFilePathFromResources(filePath)
```

Comment:
[SUGGESTION] Try to merge the two methods `textFilePathFromResources` and `loadTextFile` together. You should be able to do something like:
```
import scala.io.Source
val readmeText : Iterator[String] = Source.fromResource("readme.txt").getLines
```